GVRf/Framework/framework/src/main/res/raw/vertex_template_multitex.vsh

```diff
@@ -5,6 +5,7 @@ uniform mat4 u_view_[2];
 uniform mat4 u_mvp_[2];
```

Comment:
Why not update vertex_template.vsh also?

Reply:
we are not using vertex_template anymore. all phong shaders are using multitex ones by default.